pkg/config/config.go

```diff
@@ -83,37 +89,34 @@ func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
 	cfg := ini.Empty()
 	err := ini.ReflectFrom(cfg, nsxOperatorConfig)
 	if err != nil {
-		log.Error(err, "failed to load default NSX Operator configuration")
```

Comment:
Is it because error can be printed in upper layer, so you remove them all?

Reply:
@dantingl  Exists mutual dependency between config and log, so let's firstly get log level from config and then use log afterwords.